core/arch/arm/mm/core_mmu_lpae.c

```diff
@@ -841,6 +859,56 @@ bool core_mmu_entry_to_finer_grained(struct core_mmu_table_info *tbl_info,
 	return true;
 }
 
+/*
+ * For a table entry that points to a table - allocate and copy to
+ * a new pointed table. Does this only for the requested entry,
+ * without going deeper into the pointed table entries.
+ *
+ * A success is returned for non-table entries, as nothing to do there.
+ */
+static bool core_mmu_entry_copy(struct core_mmu_table_info *tbl_info,
+				unsigned int idx)
+{
+	uint64_t *orig_table = NULL;
+	uint64_t *new_table = NULL;
+	uint64_t *entry = NULL;
+	struct mmu_partition *prtn = NULL;
+
+#ifdef CFG_VIRTUALIZATION
+	prtn = tbl_info->prtn;
+#else
+	prtn = &default_partition;
+#endif
```

Comment:
prefer `if (IS_ENABLED(CFG_VIRTUALIZATION))`

Reply:
Unfortunately we can't do that, as existence of `prtn` inside `typeof(tbl_info)` is ifdefed already.